--- fb-athenaeum-0.1.9/libs/facebook_api/facebookapi_php5_restlib.php~	2009-12-21 23:15:12.000000000 +0200
+++ fb-athenaeum-0.1.9/libs/facebook_api/facebookapi_php5_restlib.php	2010-06-03 16:43:50.067484407 +0300
@@ -32,8 +32,6 @@
 // +---------------------------------------------------------------------------+
 //
 
-include_once 'jsonwrapper/jsonwrapper.php';
-
 class FacebookRestClient {
   public $secret;
   public $session_key;
